### PR TITLE
Asman Umbetov Tasks 31-40

### PR DESCRIPTION
**Pull request by fastndead**

### CODE REVIEW

File: tasks/33_array-plus-array/array-plus-array.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function arrayPlusArray(arr1, arr2) {
+  const result = [];
+  for (let i = 0; i < arr1.length || i < arr2.length; i++) {
+    let numberFromArr1 = arr1[i] ? arr1[i] : 0;
```
Comment (vovcyan) [inline tasks/33_array-plus-array/array-plus-array.js]: const

File: tasks/36_row-weight/row-weight.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export function rowWeights(arr) {
+  let flag = true;
+  return arr.reduce((acc, current) => {
+      flag? acc[0] += current : acc[1] += current;
```
Comment (vovcyan) [inline tasks/36_row-weight/row-weight.js]: What is it?
Rewrite it with If condition

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by vovcyan — APPROVED** [review]

File: tasks/32_numbers-to-reversed-array/numbers-to-reversed-array.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function numberToReversedArray(number) {
+  return number
+      .toString()
+      .split('')
+      .reverse()
+      .map((item) => parseInt(item));
```
Comment (alexkond) [inline tasks/32_numbers-to-reversed-array/numbers-to-reversed-array.js:6]: You could just pass `parseInt` function to `.map()`

File: tasks/37_min-number-to-prime/min-number-to-prime.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+function isPrime(number) {
+  let isPrime = number !== 1;
+  for (let j = 2; j< Math.sqrt(number)+1; j++) {
+    if (number % j === 0) {
+      isPrime = false;
+      break;
+    }
+  }
+  if (isPrime) {
```
Comment (savchukm) [inline tasks/37_min-number-to-prime/min-number-to-prime.js:18]: return `!!isPrime` is better solution here

**Review by savchukm — APPROVED** [review]

**Review by alexkond — APPROVED** [review]

**Pull request merged**